scheduler/config-k8s.edn

```diff
@@ -49,6 +59,9 @@
            :user-metrics-interval-seconds 60}
  :nrepl {:enabled? true
          :port #config/env-int "COOK_NREPL_PORT"}
+ :pools {:default "k8s-alpha"
+         :job-resource-adjustment {:adjust-job-resources-fn cook.kubernetes.api/adjust-job-resources-fn
```

Comment:
Is this for adding resources for running the filesever sidecar?

Reply:
yes